pytest_splunk_addon/standard_lib/app_test_generator.py

```diff
@@ -70,17 +70,39 @@ def generate_tests(self, fixture):
             yield from self.dedup_tests(
                 self.cim_test_generator.generate_tests(fixture), fixture
             )
-        elif fixture.startswith("splunk_indextime_fields"):
+        elif fixture.startswith("splunk_indextime"):
             # TODO: What should be the id of the test case?
             # Sourcetype + Host + Key field + _count
             addon_path = self.pytest_config.getoption("splunk_app")
             sample_generator = SampleGenerator(addon_path)
```

Comment:
Please move this logic to indextime_test_generator.  
The practice followed in searchtime is that we only call `generate_tests` method of a specific test generator from here. 
The partition was created to save complexity in a single class/method.